assets/app/views/_templateopt.html

```diff
@@ -8,16 +8,37 @@
       <a class="action action-inline" href="javascript:;" ng-click="toggleOptionsExpanded()" ng-hide="optionsExpanded"><i class="pficon pficon-edit"></i> Edit Parameters</a>
     </div>
   </div>
-  <div class="form-group options" ng-repeat="parameter in template.parameters" ng-show="optionsExpanded">
-    <label for="param-{{$index}}" title="{{ parameter.description }}">{{parameter.name}}</label>
-    <input id="param-{{$index}}" class="form-control" type="text" placeholder="{{ parameter | parameterPlaceholder }}" ng-model="parameter.value">
+  <div
+      class="form-group options"
+      ng-repeat="parameter in template.parameters"
+      ng-show="optionsExpanded"
+      ng-init="paramID = 'param-' + $index">
+    <label ng-attr-for="{{paramID}}">{{parameter.name}}<span ng-if="parameter.required"> *</span></label>
```

Comment:
Ugh, negative margin doesn't work with labels that use class `truncate`. The asterisk is hidden.